pyaedt/modeler/Object3d.py

```diff
@@ -643,6 +643,36 @@ def faces(self):
                 faces.append(FacePrimitive(self, face))
         return faces
 
+    @property
+    def top_face(self):
+        """
+        Return the Top Face on the Z Direction of the object
+
+        Returns
+        -------
+        FacePrimitive
+        """
+        result = []
+        for i in self.faces:
+            result.append((float(i.center[2]), i))
```

Comment:
```suggestion
        result = [(float(face.center[2]), face) for face in self.faces]
```
I think this is still quite readable, but I changed `i` to face to make it more so.